src/window/editor/scenario_condition_edit.c

```diff
@@ -52,46 +68,52 @@ static void draw_background(void)
 
 static void draw_foreground(void)
 {
+    data.xml_info = scenario_events_parameter_data_get_conditions_xml_attributes(data.condition->type);
```

Comment:
Can't this line go to `draw_brackground` instead?

If you keep it here this calculation is done 60 times per second, which seems overkill.

Reply:
Done. scenario_action_edit also updated.